Sources/NIOTransportServices/NIOTSConnectionChannel.swift

```diff
@@ -891,4 +862,30 @@ fileprivate extension ChannelState where ActiveSubstate == NIOTSConnectionChanne
         }
     }
 }
+
+@available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
+extension NIOTSConnectionChannel {
+    internal struct SynchronousOptions: NIOSynchronousChannelOptions {
+        @usableFromInline
+        internal let channel: NIOTSConnectionChannel
+
+        fileprivate init(channel: NIOTSConnectionChannel) {
+            self.channel = channel
+        }
+
+        @inlinable
+        public func setOption<Option: ChannelOption>(_ option: Option, value: Option.Value) throws {
+            try self.channel.setOption0(option: option, value: value)
+        }
+
+        @inlinable
```

Comment:
I think you can remove these inlinable definitions: this type is only accessed through an existential container so it can't be inlined through.